docs/usage/control_plane_migration.md

```diff
@@ -10,23 +10,24 @@ Also, the involved Seeds need to have enabled BackupBuckets.
 
 `ShootState` is an API resource which stores non-reconstructible state and data required to completely recreate a `Shoot`'s control plane on a new `Seed`.  The `ShootState` resource is created on `Shoot` creation in its `Project` namespace and the required state/data is persisted during `Shoot` creation or reconciliation.
 
-## `shoots/binding` subresource
-
-The `shoots/binding` subresource is used to bind a shoot to a seed. On creation of shoot clusters, the scheduler creates such a binding automatically. 
-```yaml
-apiVersion: core.gardener.cloud/v1beta1
-kind: Binding
-metadata:
-  name: <name>
-spec:
-  target:
-    apiVersion: core.gardener.cloud/v1beta1
-    kind: "Seed"
-    name: <seed-name>
-```
-The CreateBinding call just alters the spec.seedName and the `binding` will not be persisted anywhere.
+## Shoot Control Plane Migration
+
+Triggering the migration is done by changing the `Shoot`'s `.spec.seedName` to a `Seed` that differs from the `.status.seedName`, we call this `Seed` `"Destination Seed"`. If the Destination `Seed` does not have a backup and restore configuration, the change to `spec.seedName` is rejected. Additionally, this Seed must not be set for deletion and must be healthy.
+
+If the `Shoot` has different `.spec.seedName` and `.status.seedName` a process is started to prepare the Control Plane for migration:
+
+1. `.status.lastOperation` is changed to `Migrate`.
+2. Kubernetes API Server is stopped and the extension resources are annotated with `gardener.cloud/operation=migrate`.
+3. Full snapshot of the ETCD is created and terminating of the Control Plane in the `Source Seed` is initiated.
+
+If the process is successful, we update the status of the `Shoot` by setting the `.status.seedName` to the null value. That way, a restoration is triggered in the `Destination Seed` and `.status.lastOperation` is changed to `Restore`. The control plane migration is completed when the `Restore` operation has completed successfully.
+
+By default the shoot's etcd backups will continue to be uploaded to the `BackupBucket` of the `Source Seed` after control plane migration has finished. If you want the etcd backups to instead be uploaded to the `BackupBucket` of the `Destination Seed`, you have to enable the `CopyEtcdBackupsDuringControlPlaneMigration` feature gate on the `gardenlet` by adding the following command flag: `--feature-gates=CopyEtcdBackupsDuringControlPlaneMigration=true`. Note that this will also move the existing etcd backups to the `BackupBucket` of the `Destination Seed` during the control plane migration process.
+
+## Triggering the migration
+
+For controlplane migration, operators with necessary RBAC can request a [`shoot/binding`](../concepts/scheduler.md#shootsbinding-subresource), with the following commands:
```

Comment:
Probably it makes sense to move this "how to do it manually" to the scheduler doc as well and just refer to it from here.

Reply:
Does https://github.com/gardener/gardener/pull/6018/commits/7134bf33853fc2c9d790a71675596740684eea7f look fine?